js/client-chat.js

```diff
@@ -567,11 +567,20 @@
 					for (var i=0, len=targets.length; i<len; i++) {
 						targets[i] = targets[i].trim();
 					}
-					switch (targets[0]) {
+					switch (targets.shift()) {
 					case 'add':
-						for (var i=1, len=targets.length; i<len; i++) {
-							highlights.push(targets[i].trim());
-						}
```

Comment:
Ah. ``targets[i].trim()`` is already applied before the switch statement, though